src/index.css

```diff
@@ -90,6 +91,11 @@
     .post-content {
         overflow-wrap: break-word;
     }
+
+    svg.mui {
+        height: 1rem;
+        width: 1rem;
+    }
```

Comment:
please give clear classnames

Reply:
This was to style the mui (Material UI) svg icons. That's why the short `mui`